### ISSUE TITLE
unable to install on Windows 11 ARM64 VM

### ISSUE THREAD

**Issue by TartCart** [GyulyVGC/sniffnet#49]
Having an issue with install for a windows virtual machine on an mac m1 pro. trying to duplicate what the install would be like on a regular x64 machine.
I have tried both cargo and the provided MSI.

in the installation readme instructions, I have installed NPcap and downloaded the required npcap SDK but am unsure how to add the "Add the SDK's /Lib or /Lib/x64 folder to your LIB environment variable." 
Tried googling to find some direction but haven't found anything useful.
<img width="504" alt="Screenshot 2022-12-31 at 4 49 31 PM" src="https://user-images.githubusercontent.com/67134507/210129787-6ccfb34d-4690-4820-83da-a684ad76d999.png">
<img width="962" alt="Screenshot 2022-12-31 at 4 49 22 PM" src="https://user-images.githubusercontent.com/67134507/210129788-cf57ebd4-5f15-4944-aa50-27b510f49f37.png">

Attached screenshots of errors when installing with Rust and running the sniffnet.exe after installing with the provided MSI.

Any direction would be greatly appreciated. thanks

**Comment by GyulyVGC** [comment]
[Here](https://www.aomeitech.com/windows-tips/set-windows-11-environment-variables.html) you can find how to add new environment variables. In your case, you have to specify the Lib/x64 path inside the SDK you have downloaded.

You will not be able to build it with cargo because you don’t have VS code installed. Let me know if it works with the MSI after having modified your environment variables. Just to be sure, add it both to your and to system environment variables.

**Comment by GyulyVGC** [comment]
I also suggest you to follow [this](https://www.drivereasy.com/knowledge/how-to-fix-the-application-is-unable-to-start-correctly-0xc00007b-error/) guide, in particular try performing steps 2, 3 and 5.

**Comment by TartCart** [comment]
Thanks for the prompt reply -
I added a user and system environment variable named "LIB" to the path of the SDK's "Lib" folder. (I also tried modifying to the x64 directory). I am getting the same error when running the MSI, I reinstalled and ran as admin. 
Tho, I have had issues in the past with Windows ARM and running programs. I will try with x64 bit machine when I return home from traveling. thanks again.
